src/components/PopupWithForm.js

```diff
@@ -35,4 +30,12 @@ export default class PopupWithForm extends Popup {
       this._popupForm.reset();
```

Comment:
Needs correction: you should reset the form only after the successful submission in `then` block in index.js otherwise the form  will become empty even if an error occurred

Reply:
Still needs correction